components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.enrich.ui/src/main/java/org/wso2/carbon/mediator/enrich/ui/EnrichMediator.java

```diff
@@ -57,26 +56,21 @@
     private String sourceInlineXML = "";
     private String inlineSourceRegKey = "";
 
-
     private SynapseXPath targetExpression = null;
     private String targetProperty = "";
 
-
     public String getSourceType() {
```

Comment:
Shall we add doc comments to all the methods?

Reply:
Done